apispec/openapi.json

```diff
@@ -5577,7 +5577,7 @@
       "Accept-Encoding": {
         "name": "Accept-Encoding",
         "in": "header",
-        "description": "Indicates the compression format of response data, supports \"gzip\" and \"uncompressed\", default value is \"uncompressed\".",
+        "description": "Indicates the compression format of response data, supports \"gzip\" and \"uncompressed\". The default value is \"uncompressed\".",
```

Comment:
Suggestion:  "Indicates the compression format of response data, either \"gzip\" or \"uncompressed\".